src/itkOMEZarrNGFFImageIO.cxx

```diff
@@ -807,8 +812,13 @@ OMEZarrNGFFImageIO::Write(const void * buffer)
     itkExceptionMacro("Unsupported component type: " << GetComponentTypeAsString(this->GetComponentType()));
   }
 
-  // Create a new context to close the open zip handles
-  tsContext = tensorstore::Context::Default();
+  if (m_FileName.substr(m_FileName.size() - 4) == ".zip" || m_FileName.substr(m_FileName.size() - 7) == ".memory")
+  {
+    // Attempt to read a non-existent file from the in-memory zip to close the current one
```

Comment:
Is this a workaround for an existing Tensorstore issue?

Reply:
I don't know whether the issue is in tensorstore, or in our zip addition to it.